tests/io/v3/was/test_folders.py

```diff
@@ -0,0 +1,55 @@
+import responses
+
+WAS_FOLDERS_BASE_URL = 'https://cloud.tenable.com/api/v3/was/folders'
+SAMPLE_FOLDER_ID = '178fe279-4e37-49ee-a5dc-8a447dd7043a'
+SAMPLE_FOLDER = {
+    'folder_id': SAMPLE_FOLDER_ID,
+    'name': 'Folder name'
+}
+
+
+@responses.activate
+def test_create(api):
+    responses.add(
+        responses.POST,
+        WAS_FOLDERS_BASE_URL,
+        json=SAMPLE_FOLDER
+    )
+    folder = api.v3.was.folders.create(SAMPLE_FOLDER['name'])
+    assert isinstance(folder, dict)
+    assert folder['folder_id'] == SAMPLE_FOLDER_ID
+
+
+@responses.activate
+def test_delete(api):
+    responses.add(
```

Comment:
Please add the doc strings in all tests.

Reply:
added doc strings